        <html><head></head><body>
        <h1>Report for unetbootin on toolchain
            <a href="index.html">vanilla</a>
        </h1>
        <p>Return code: 1</p>
        <p>Time to run: 00:00:34</p>
        <p>Log output:</p>
        <code><pre><docker run --rm -v /tuscan_data --volumes-from tuscan_data -v /sources --volumes-from sources -v /var/cache/pacman/pkg --volumes-from pkg_cache_vanilla -v logs:/logs make_package_container --sources-directory /sources --shared-directory /tuscan_data unetbootin
:: Synchronizing package databases...
downloading repo.db...
downloading core.db...
downloading extra.db...
downloading community.db...
Found 2139 packages in cache
Found permanent source directory in /sources/unetbootin
Traceback (most recent call last):
  File "/build/make_package.py", line 215, in <module>
    main()
  File "/build/make_package.py", line 212, in main
    result = copy_and_build(args)
  File "/build/make_package.py", line 118, in copy_and_build
    copytree(permanent_source_dir, build_dir)
  File "/usr/lib/python3.5/shutil.py", line 353, in copytree
    raise Error(errors)
shutil.Error: [('/sources/unetbootin/src/unetbootin/src/booteder/nsis', '/tmp/unetbootin/src/unetbootin/src/booteder/nsis', "[Errno 2] No such file or directory: '/sources/unetbootin/src/unetbootin/src/booteder/nsis'"), ('/sources/unetbootin/src/unetbootin/src/downlder/nsis', '/tmp/unetbootin/src/unetbootin/src/downlder/nsis', "[Errno 2] No such file or directory: '/sources/unetbootin/src/unetbootin/src/downlder/nsis'")]
Error deleting container: Error response from daemon: Driver aufs failed to remove root filesystem dc8131e73e25fa5579b126033ef6ed6db53ba0aed2d92532e34de110cc60cc0f: rename /usr/local/google/home/khazem/docker/aufs/diff/dc8131e73e25fa5579b126033ef6ed6db53ba0aed2d92532e34de110cc60cc0f /usr/local/google/home/khazem/docker/aufs/diff/dc8131e73e25fa5579b126033ef6ed6db53ba0aed2d92532e34de110cc60cc0f-removing: device or resource busy></pre></code>
        </body></html>
